borg/testsuite/repository.py

```diff
@@ -420,7 +420,28 @@ def open(self, create=False):
                                 exclusive=True, create=create)
 
     def test_invalid_rpc(self):
-        self.assert_raises(InvalidRPCMethod, lambda: self.repository.call('__init__', None))
+        try:
+            self.repository.call('__init__', None)
```

Comment:
ok, this was already strange before your change, but maybe we could fix it anyway:

``__init__`` rather looks like a valid rpc method. if we want something invalid, how about 'something_invalid'?